tests/MongoDB.Driver.Tests/PipelineDefinitionBuilderTests.cs

```diff
@@ -114,6 +128,11 @@ public void Merge_should_add_expected_stage()
             stages[0].Should().Be("{ $merge : { into : { db : 'database', coll : 'collection' } } }");
         }
 
+
```

Comment:
empty line

Reply:
can this class be simply replaced by `object`?

Reply:
I agree that creating a new class is confusing, specially when given a random name like Starship that has nothing to do with what's actually being tested.

Reply:
My intent was to have an example class like "Pet" or "Person"' but this is now moot because I've used said "Pet" and "Person" classes from the pre-existing testse :)